docs/spec_draft.md

```diff
@@ -397,6 +458,58 @@ lexicographic comparison on the (real, imaginary) pairs.
 
 [Back to Ops](#index-of-documented-ops)
 
+## stablehlo.negate
+
+`stablehlo.negate(operand) -> result`
+
+### Semantics
+
+Performs element-wise negation of `operand` tensor and produces a `result`
+tensor. For floating-point element types, implements the `nagate` operation from
+the IEEE-754 specification. For integer types, performs the [two's complement](https://en.wikipedia.org/wiki/Two's_complement)
+operation.
```

Comment:
Above, we're already saying that signed integers are using two’s complement representation, so it looks like for them this sentence redundant. Is this written for unsigned integers?

Perhaps reformulate to provide more details, e.g. "For signed integer types, performs the regular negation operation, where the negation of $-2^{n-1}$ is implementation defined and one of the following: <...>. For unsigned integer types, bitcasts to the corresponding signed integer type, performs the regular negation operation and bitcasts back to the original unsigned integer type"?

Reply:
I wrote them for both given that in two's complement arithmetic is same for the fundamental  operations  like add/sub/ etc for unsigned binary numbers (as long as the inputs are represented in the same number of bits as the output, and any overflow beyond those bits is discarded from the result).

I like your suggestion which is definitely more clearer.